src/BuffCommand.cpp

```diff
@@ -50,6 +50,20 @@ class KargatumCS_BuffCOmmand : public CommandScript
 		Player* player = handler->GetSession()->GetPlayer();
 		std::string ArgStr = (char*)args;
 
+		if (sConfigMgr->GetIntDefault("BuffCommand.MinLevel", 80) > 0) {
+			int MinLevel  = sConfigMgr->GetIntDefault("BuffCommand.MinLevel", 80);
```

Comment:
maybe use `uint32`?

Reply:
Hmm. Maybe uint8? Hax a max of 255 exactly the max level on custom servers

Reply:
hm, yes